README.md

```diff
@@ -2,6 +2,7 @@
 
 [![](https://jitpack.io/v/badoo/Reaktive.svg)](https://jitpack.io/#badoo/Reaktive)
 [![](https://img.shields.io/badge/License-Apache/2.0-blue.svg)](https://github.com/badoo/Reaktive/blob/master/LICENSE)
+[![Build Status](https://travis-ci.org/badoo/Reaktive.svg?branch=master)](https://travis-ci.org/badoo/Reaktive)
```

Comment:
Cool! Don't you think it will look better between version and version badges? :-D

Reply:
Between version and license?

Reply:
Yes)))